### PR TITLE
add load_desktop_reports command

### PR DESCRIPTION
**Pull request by vjeffrey**
### :nut_and_bolt: Description: What code changed, and why?
added command for all the desktop needs
- loads a few nodes then sets their last check in time to a long ago
- loads a few desktop reports, that should have all those special fields we need for the views
- loads some errored runs

### :athletic_shoe: How to Build and Test the Change
source .studio/ingest-service
load_desktop_reports
visit a2-url/desktop, see data

### CODE REVIEW

File: .studio/ingest-service [hunk 1]
```diff
@@ -328,3 +328,22 @@ function ingest_grpcurl_configure_purge() {
...
+  {
+  "script": {
+  "source": "ctx._source[\u0027checkin\u0027] = \u00272020-02-15T02:43:43Z\u0027",
+  "lang": "painless"
+  }
+  }
+  '
+  cat dev-docs/adding-data/sample-reports/desktop-report-1.json | curl --insecure -H "api-token: $(get_admin_token)" -X POST "https://a2-dev.test/data-collector/v0" -H "Content-Type: application/json" -d @-
+  cat dev-docs/adding-data/sample-reports/desktop-report-2.json | curl --insecure -H "api-token: $(get_admin_token)" -X POST "https://a2-dev.test/data-collector/v0" -H "Content-Type: application/json" -d @-
+  cat dev-docs/adding-data/sample-reports/desktop-report-3.json | curl --insecure -H "api-token: $(get_admin_token)" -X POST "https://a2-dev.test/data-collector/v0" -H "Content-Type: application/json" -d @-
```
Comment (srenatus) [inline .studio/ingest-service:347]: [nit]
```suggestion
  for file in cat dev-docs/adding-data/sample-reports/desktop-report-{1,2,3}.json; do
    curl --insecure -H "api-token: $(get_admin_token)" -X POST "https://a2-dev.test/data-collector/v0" -H "Content-Type: application/json" -d "$file"
  end
```

File: .studio/ingest-service [hunk 2]
```diff
@@ -328,3 +328,22 @@ function ingest_grpcurl_configure_purge() {
...
+  for _ in {1..30} ; do   m=$((RANDOM % 4));   n=$((RANDOM % 10));   generate_chef_run_failure_example |     jq --arg msg "Error $n occurred" --arg type "Chef::ExampleError$m" '.error.message = $msg | .error.class = $type' |     send_chef_data_raw; done
```
Comment (srenatus) [inline .studio/ingest-service:348]:
```suggestion
  for _ in {1..30}; do
    m=$((RANDOM % 4))
    n=$((RANDOM % 10))
    generate_chef_run_failure_example |\
      jq --arg msg "Error $n occurred" \
        --arg type "Chef::ExampleError$m" \
        '.error.message = $msg | .error.class = $type' |\
      send_chef_data_raw
  done
```

File: .studio/ingest-service (hunk 1 above)
Comment (srenatus) [reply .studio/ingest-service:347]: I also think we've got env vars for the API endpoint (`a2-dev.test` hardcoded here) but I don't know your use case here.

Comment (vjeffrey) [reply .studio/ingest-service:347]: oooh thanks! and good point: if i make the url/token not hardcoded we could use this for populating the acceptance instances on demand

**Comment by susanev** [comment]
im getting this when trying to source the ingest-service

```
bash: .studio/ingest-service: line 349: syntax error near unexpected token `}'
bash: .studio/ingest-service: line 349: `}'
```

File: .studio/ingest-service [hunk 3]
```diff
@@ -328,3 +328,30 @@ function ingest_grpcurl_configure_purge() {
...
+  {
+  "script": {
+  "source": "ctx._source[\u0027checkin\u0027] = \u00272020-02-15T02:43:43Z\u0027",
+  "lang": "painless"
+  }
+  }
+  '
+  for file in cat dev-docs/adding-data/sample-reports/desktop-report-{1,2,3}.json; do
+    curl --insecure -H "api-token: $(get_admin_token)" -X POST "https://a2-dev.test/data-collector/v0" -H "Content-Type: application/json" -d "$file"
+  end
```
Comment (stevendanna) [inline .studio/ingest-service:347]:
```suggestion
  done
```

**Comment by vjeffrey** [comment]
ya sorry im fixing it up post-commit-add

**Comment by vjeffrey** [comment]
alright should be working now @susanev

**Comment by vjeffrey** [comment]
ill do a follow up to take params for the url and token in another pr

**Pull request merged**